vin/config/initializers/db_initial_values.rb

```diff
@@ -1,11 +0,0 @@
-#encoding: utf-8
-
-data_folder = Rails.root.join("lib", "data")
-
-# illegal = JSON.parse(File.read(File.join(data_folder, "illegal_states.json")))
-
-wines = JSON.parse(File.read(File.join(data_folder, "wines.json"))).flat_map { |k,wine| wine }
-
-wines.each do |wine|
-  Wine.create(wine)
```

Comment:
Interesting read for this http://edgeguides.rubyonrails.org/active_record_migrations.html#migrations-and-seed-data

Yeah, Rails has it all figured out